Administrator Guide/NFS-Ganesha GlusterFS Integration.md

```diff
@@ -301,3 +143,13 @@ GLUSTER
 
    - It is tested and verified with RHEL 6.5 , fedora 20, fedora 21 nfs clients. It is always better to use latest nfs-clients
 
+## Tunables for NFS-Ganesha
+- NFS_CORE_PARAM {} in ganesha.conf
+	- Large no of clients: RPC_Max_Connections (default is 1000)
```

Comment:
What's the goal here ?

Do you intend to say to the user that if there are large number of clients, increase the RPC tunable.

It's not clear.